cli/cli/helpers/engine_manager/engine_existence_guarantor.go

```diff
@@ -65,6 +63,9 @@ type engineExistenceGuarantor struct {
 	//TODO This is a temporary hack we should remove it when centralized logs be implemented in the KubernetesBackend
 	kurtosisClusterType resolved_config.KurtosisClusterType
 
+	// Engine on bastion?
+	onBastionHost bool
```

Comment:
Thinking more about it, I am wondering if I should supply this information using the `engine_server_launcher.KurtosisBackendConfigSupplier`.

Reply:
That would make sense to me yes. Keeping all the backend config info at the same place is cleaner, and this boolean looks like a piece of config.
But thinking more about it, do we need this flag? From now, if the context is remote, then the engine will necessarily be on the bastion right? So, this flag to me is duplicate of kurtosisContext.IsRemote. Wondering if we could merge those together instead

Reply:
This config supplier returns a config which gets mapped to a local backend config object which made things confusing for the bastion host flag.  I kept things as is for now.